css/css-fonts/variations/at-font-face-font-matching.html

```diff
@@ -79,36 +83,67 @@
         ];
 
         descriptorPriorityCases.forEach(function (testCase) {
-            promise_test(
-                assert => once_fonts_are_ready.then(assert => {
-                    verifyFont("descriptorPriorityTest", testCase.weight, testCase.style, testCase.stretch, testCase.expectedFamily);
-                }),
+            promise_test(() => {
+                return once_fonts_are_ready
+                    .then(() => verifyFont("descriptorPriorityTest", testCase.weight, testCase.style, testCase.stretch, testCase.expectedFamily));
+                },
                 "Descriptor mathcing priority: " + testCase.description
```

Comment:
BTW, while you're here, maybe you could also fix the typo? s/mathcing/matching/

Reply:
Sure